src/Microsoft.ML.FastTree/BoostingFastTree.cs

```diff
@@ -40,6 +39,16 @@ protected BoostingFastTreeTrainerBase(IHostEnvironment env, TArgs args, SchemaSh
             }
         }
 
+        protected BoostingFastTreeTrainerBase(IHostEnvironment env,
+            SchemaShape.Column label,
+            string featureColumn,
+            string weightColumn,
+            string groupIdColumn,
+            TArgs advancedSettings)
```

Comment:
>TArgs advancedSettings [](start = 12, length = 22)

Hi @Abhishek Goswami, thanks for looking into this issue. Your change will also fix the problem described in issue #2019 ([https://github.com/dotnet/machinelearning/issues/2019](https://github.com/dotnet/machinelearning/issues/2019)).

Reply:
Thanks for the info.  Will reference #2019 in my PR  so it gets closed upon merging this PR

---
In reply to: [245743197](https://github.com/dotnet/machinelearning/pull/2047#discussion_r245743197) [](ancestors = 245743197)